src/app/layouts/default/footer/footer.js

```diff
@@ -127,27 +127,11 @@ function Footer({
     );
 }
 
-function OsanoToggle() {
-    const osano = typeof window === 'undefined' ? undefined : window.Osano;
-    const showOsanoDrawer = React.useCallback(
-        (e) => {
-            e.preventDefault();
-            osano.cm.showDrawer('osano-cm-dom-info-dialog-open');
-        },
-        [osano]
-    );
-
-    if (osano === undefined || osano.cm.mode === 'debug') {
-        document.body.classList.remove('hide-osano');
-        return null;
-    }
-
-    document.body.classList.add('hide-osano');
+function CookieYesToggle() {
```

Comment:
I should move this to ui-components and use it in Rex

Reply:
It wasn't done in a shared way before because Roy didn't want to add styled components to osweb

You could pretty much copy/paste though. Maybe rex and assignable could share a component